src/containers/App.js

```diff
@@ -22,6 +23,7 @@ function App() {
             `&maxResults=30&key=AIzaSyDYgrEqAsmIyoRmLzx6rNDSAcGPubpDJ-Q`)
       .then(response => response.json())
       .then(json => {
+        setLoaderVisibility(true);
```

Comment:
what happens if the request fails? 
use finally block to stop loader

Reply:
I will try to do it